from unittest import TestCase


class Test(TestCase):
    def test_main(self):
        assert self.fail()



